lib/makara/proxy.rb

```diff
@@ -248,28 +243,28 @@ def hijacked
     end
 
 
-    def previously_stuck_to_master?
-      @sticky && Makara::Context.previously_stuck?(@id)
+    def stuck_to_master?
+      sticky? && Makara::Context.stuck?(@id)
     end
 
-
-    def stick_to_master(method_name, args, write_to_cache = true)
```

Comment:
I think it's not necessary here, because in all cases we call `stick_to_master`, which is private to the proxy, it's because we came across some action that made us to use `master`, and in that case we always want to persist the value for the `master_ttl` duration.